source/server/index.js

```diff
@@ -1,5 +1,11 @@
 import createCuid from '../index.js';
-const fingerprint = require('node-fingerprint')();
+
+// workaround for webpack's process polyfill
+try {
+  process.pid = process.pid || Date.now();
```

Comment:
Could you do this without `try/catch`, and without assigning to a system resource?

`const pid = process && process.pid || Date.now();`